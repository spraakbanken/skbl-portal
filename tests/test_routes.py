import pytest


@pytest.mark.parametrize("route", ["/sv", "/en", "/sv/om-skbl", "/en/about-skbl"])
def test_routes(client, route: str, snapshot):
    response = client.get(route)

    assert response.status_code == 200  # noqa: PLR2004
    response_text = response.get_data(as_text=True)
    assert response_text[:100] == snapshot
