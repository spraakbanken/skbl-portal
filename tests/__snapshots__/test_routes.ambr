# serializer version: 1
# name: test_routes[/en/about-skbl]
  '''
  <!doctype html>
  <html lang="en">
    <head>
      
      <title>skbl.se - About SKBL</title>
      <meta nam
  '''
# ---
# name: test_routes[/en]
  '''
  <!doctype html>
  <html lang="en">
    <head>
      
      <title>skbl.se - Svenskt kvinnobiografiskt lexiko
  '''
# ---
# name: test_routes[/sv/om-skbl]
  '''
  <!doctype html>
  <html lang="sv">
    <head>
      
      <title>skbl.se - Om SKBL</title>
      <meta name="
  '''
# ---
# name: test_routes[/sv]
  '''
  <!doctype html>
  <html lang="sv">
    <head>
      
      <title>skbl.se - Svenskt kvinnobiografiskt lexiko
  '''
# ---
